src/main/java/com/google/api/codegen/transformer/ruby/RubySurfaceNamer.java

```diff
@@ -48,6 +49,12 @@ public String getFormatFunctionName(CollectionConfig collectionConfig) {
     return staticFunctionName(Name.from(collectionConfig.getEntityName(), "path"));
   }
 
+  public String getGrpcMethodName(Method method) {
+    // This might seem silly, but it makes clear what we're dealing with (upper camel).
+    // This is language-independent because of gRPC conventions.
+    return Name.upperCamel(method.getSimpleName()).toLowerUnderscore();
```

Comment:
The comment doesn't make sense in this context, so remove it. Also, why is this lower underscore? Does the generated grpc layer for Ruby use lower underscore?